spec/models/organization_spec.rb

```diff
@@ -8,15 +8,44 @@
   it { should validate_uniqueness_of(:login) }
 
   describe 'load_organizations' do
+    let(:whitelisted_organisations) { ['test-organization'] }
+    before do
+      stub_const("Organization::WHITELISTED_ORGS", whitelisted_organisations)
+    end
+
     context 'creates new organization' do
```

Comment:
Why do you even need this context? I suggest dropping it.

Reply:
I mean, just put `it` block at the top level — they work outside of any contexts as well.